use-cases/predictive_maintenance/1_dataprep_dw_job_predmaint.ipynb

```diff
@@ -4,18 +4,45 @@
    "cell_type": "markdown",
```

Comment:
<p>For all notebook titles/introductions mentioning "Data Wrangler", let's specifically say <code>SageMaker Data Wrangler</code> because there is also the AWS Data Wrangler Python library (https://github.com/awslabs/aws-data-wrangler), so we can disambiguate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p><p> </p>